explainer/Explainer.md

```diff
@@ -1,9 +1,369 @@
-TD, Discovery, and Profile explainers to add detail and provide use cases, justifications, requirements,
-and examples for their specific deliverables.
+## Introduction
+WoT Discovery builds upon existing network service and IoT discovery mechanisms
+but also provides secure RESTful web service APIs for searching WoT Thing Description metadata.
+Discovery has to simultaneously satisfy many objectives which are in partial conflict:
+it needs to make data easily available, but preserve privacy; it needs to work for small
+devices and installations but also for large systems; it needs to work on small LANs and
+on the internet; it needs to enable semantic search without burdening implementations; and it 
+needs to support new, self-describing devices but also work with brownfield devices that
+cannot be modified and so can only be described externally.
 
-See above PR for links to instructions for writing explainers.
+The WoT Discovery specification attempts to meet these challenges by defining
+a multistage discovery process with authorization checkpoints to meet privacy and
+security goals; by using scalable RESTful API designs; by supporting a variety of
+first-contact mechanisms suitable for different scales; and by providing optional
+semantic search and self-description capabilities.
+
+There are still a number of limitation in the current specification, which
+primarily focuses on HTTP and had to defer some important goals, such as
+geolocation queries.  However, the specification attempts to address the above
+core issues and is intended to serve as a foundation for future work while
+enabling, in its current form, use cases supporting IoT integration
+and access to public IoT services.
+
+## Relationship to Other WoT Deliverables
+For a general introduction to the Web of Things (WoT) Architecture, including a description
+of all normative and informative deliverables, please see the 
+[WoT Architecture explainer](https://github.com/w3c/wot-architecture/blob/main/explainer/explainer11.md)
+and the [WoT Architecture document](https://w3c.github.io/wot-architecture/).
+
+Since the purpose of the WoT Discovery deliverable is to normatively
+define a mechanism to distribute WoT Thing Descriptions,
+the [WoT Thing Description 1.1 document](https://w3c.github.io/wot-thing-description/) is
+especially relevant.
+The WoT Discovery specification also includes a Thing Model,
+which normatively defines many aspects of the Thing Description Directory service specified
+as part of the discovery process.
+Thing Models, as defined in the Thing Description 1.1 document,
+are a format for describing classes of Things.
+We model Thing Description Directories as a special kind of Thing.
+
+## Use Cases and Requirements
+In general, WoT Use Cases and Requirements have been collected from a large number 
+of participants and are collected in the 
+[WoT Use Cases and Requirements document](https://w3c.github.io/wot-usecases/).
```

Comment:
```suggestion
[WoT Use Cases and Requirements document](https://www.w3.org/wot-usecases/).
```
https://www.w3.org/wot-usecases/ is not found for some reason!

Reply:
"https://www.w3.org/TR/wot-usecases/" ?